Makefile

```diff
@@ -417,34 +417,43 @@ deploy-onprem:
 
 deploy-on-openshift-ci:
 	export PERSISTENT_STORAGE="False" && export TARGET='oc' && export GENERATE_CRD='false' && unset GOFLAGS && \
-	$(MAKE) ci-deploy-for-subsystem
+	$(MAKE) deploy-test
 	oc get pods
 
 ########
 # Test #
 ########
 
-subsystem-run: test subsystem-clean
-
-subsystem-run-kube-api: enable-kube-api-for-subsystem test-kube-api subsystem-clean
-
 test:
 	$(MAKE) _run_subsystem_test AUTH_TYPE=rhsso ENABLE_ORG_TENANCY=true ENABLE_ORG_BASED_FEATURE_GATES=true
 
 test-kube-api:
-	$(MAKE) _run_subsystem_test AUTH_TYPE=local ENABLE_KUBE_API=true FOCUS="$(or ${FOCUS},kube-api)"
+	$(MAKE) _run_subsystem_test AUTH_TYPE=local FOCUS="$(or ${FOCUS},kube-api)"
 
-# An alias for the test target
-subsystem-test: test
+# execute on the host, without skipper
+subsystem-test:
+	TARGET=kind skipper $(MAKE) test
 
-# An alias for the test-kube-api target
-subsystem-test-kube-api: test-kube-api
+# execute on the host, without skipper
+subsystem-test-kube-api:
+	TARGET=kind skipper $(MAKE) test-kube-api
```

Comment:
why skipper command has been introduced?
Can't this be executed `skipper make subsystem-test` ?

Reply:
you mean without skipper ?

Reply:
skipper supports makefile target, why are we executing skipper within makefile targets? I think it should be the other way round, unless we have some mixed approach (i.e. within one target, one command needs skipper and one needs to run without). WDYT?